connectors/filesystem.mdx

```diff
@@ -0,0 +1,60 @@
+---
+title: File System
+description: "Write Parquet and JSON to S3 and local filesystems"
+---
+
+Arroyo provides the capability to write Parquet and JSON files to S3 and local filesystems.
+This will produce complete files in line with the checkpointing system.
+As such, the file system sinks write all data exactly once.
+
+## Configuring the File System Sink
+
+The file system sink is currently only configurable through SQL,
+ using the [SQL DDL](sql/ddl) syntax.
+ The connector name will be `filesystem` and there are several additional options
+ 
+ ### Destination Options
+
+ | Field | Description | Required | Example |
+ | ----- | ----------- | -------- | ------- |
+ | `path` | A URI to the target destination | No | `file:///test-data/my-cool-arroyo-pipeline`, `https://s3.us-west-2.amazonaws.com/awesome-arroyo-bucket/amazing-arroyo-dir` |
```

Comment:
It's a bit confusing that these are all marked as not required—surely something is?

Is there any reason to prefer s3_bucket and s3_directory options over the more flexible URI ones? In any case, it would be good to clarify that these are different (and mutually incompatible) ways of configuring the same thing.